constants/HoppityEggLocations.json

```diff
@@ -226,7 +226,17 @@
   "coachRabbitIndex": 42,
   "maxRabbits": 457,
   "maxPrestige": 6,
-  "maxMilestoneChocolate": 700000000000,
```

Comment:
Don't replace the value here since older clients won't be able to read the new format.